### DISCUSSION TITLE
ConnectionResetError when using deepcell_service_utils.create_deepcell_output

### DISCUSSION THREAD

**Discussion by coding0lion** [angelolab/ark-analysis#1090] [Announcements]
Description:
I am a user of the ark-analysis tool. Firstly, I would like to express my appreciation for the great work that your team has done in developing this tool.
I am reaching out because I have encountered a ConnectionError while using ark-analysis. The specific error message is as follows:
This error occurs consistently, and it seems to be related to the network connection between my machine and the ark-analysis server.
I have already checked my internet connection, and I can access the ark-analysis server through a web browser without any issues. However, when I try to use ark-analysis programmatically, I encounter the mentioned ConnectionError.

Expected Behavior:
I expected the function to upload files to the DeepCell server and retrieve the results without encountering a connection error.

Steps to Reproduce:
Ran the following code: `deepcell_service_utils.create_deepcell_output(deepcell_input_dir, deepcell_output_dir, fovs=fovs, scale=rescale_factor)`

I would be grateful if you could provide guidance on how to resolve this issue. Additionally, if there are any specific logs or additional information that would be helpful for troubleshooting, please let me know, and I will gladly provide them.

**Comment by alex-l-kong** [comment]
@coding0lion this isn't a connection issue with `ark-analysis`, but rather with the Deepcell server, which hosts the Mesmer segmentation model. Every so often, it does experience outages, but they usually get resolved during the day. Are you still getting the connection error?
